windows/utils/win_get_vmtools_version_build.yml

```diff
@@ -3,20 +3,46 @@
 ---
 # Get installed VMware Tools version and build number in Windows guest OS
 # Return:
-#   vmtools_version: VMware Tools version
-#   vmtools_build_number: VMware Tools build number
+#   vmtools_version: VMware Tools version.
+#   vmtools_build_number: VMware Tools build number.
+#   vmtools_toolboxcmd_exists: Whether 'VMwareToolboxCmd.exe' exists in guest OS.
 #
 - name: "Initialize the facts of VMware Tools version and build info"
   ansible.builtin.set_fact:
     vmtools_info_from_vmtoolsd: ''
     vmtools_version: ''
     vmtools_major_version: ''
     vmtools_build_number: ''
+    vmtools_toolboxcmd_exists: false
+    vmtools_toolboxcmd_path: "C:\\Program Files\\VMware\\VMware Tools\\VMwareToolboxCmd.exe"
```

Comment:
I think here can also add a var `vmtools_install_dir: "C:\\Program Files\\VMware\\VMware Tools"`, which can replace "C:\\Program Files\\VMware\\VMware Tools" in windows test cases.

Reply:
Added, thanks.